libcloud/compute/types.py

```diff
@@ -59,112 +59,115 @@ class Provider(Type):
     """
     Defines for each of the supported providers
 
+    Non-Dummy drivers are sorted in alphabetical order. Please preserve this
+    ordering when adding new drivers.
+
     :cvar DUMMY: Example provider
+    :cvar ABIQUO: Abiquo driver
+    :cvar ALIYUN_ECS: Aliyun ECS driver.
+    :cvar AURORACOMPUTE: Aurora Compute driver.
+    :cvar AZURE: Azure driver.
+    :cvar BLUEBOX: Bluebox
+    :cvar CLOUDSIGMA: CloudSigma
+    :cvar CLOUDSIGMA_US: CloudSigma US Las Vegas
+    :cvar CLOUDSTACK: CloudStack
+    :cvar DIMENSIONDATA: Dimension Data Cloud
+    :cvar EC2_EU_WEST: Amazon AWS EU Ireland
```

Comment:
The compute driver for EC2 is now a single driver. these cvars are wrong @ktdreyer